public/components/trace_analytics/components/dashboard/latency_trend_cell.tsx

```diff
@@ -3,8 +3,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { EuiButtonIcon, EuiFlexGroup, EuiFlexItem, EuiPopover, EuiText } from '@elastic/eui';
-import _ from 'lodash';
+import {
+  EuiButtonIcon,
```

Comment:
Might have missed the Euibuttonicon here

Reply:
The `EuiButtonIcon` already has a `size`.